packages/editor/src/components/block-manager/index.js

```diff
@@ -67,8 +67,7 @@ function BlockManager( {
 						numberOfHiddenBlocks
 					) }
 					<Button
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
+						__next40pxDefaultSize
```

Comment:
<table>
<tr>
<td>Before</td>
<td>After</td>
</tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/f9bed54b-9c44-4f8a-9283-ebed98ce59c0">
</td>
    <td><img src="https://github.com/user-attachments/assets/74bae72a-db84-4103-a610-455a9a5da419">
</td>
  </tr>
</table>


Style didn't change because there are explicit styles for Reset in code. But adding `__next40pxDefaultSize` will ensure that it will not be listed in lint next time

Reply:
Note for posterity: There are no explicit height styles for this, just that  `"link"` variant Buttons have `height: auto` applied.

Reply:
😅